ports/cpptrace/portfile.cmake

```
@@ -0,0 +1,18 @@
+vcpkg_from_github(
+	OUT_SOURCE_PATH SOURCE_PATH
+	REPO jeremy-rifkin/cpptrace
+	REF 4b11b87e4d905d003d0325a53994441cc767017a
```

```suggestion
	REF "v${VERSION}"
```
(Note that this requires a change to `SHA512`.)